packages/cozy-harvest-lib/src/components/KonnectorConfiguration/Success/DriveLink.jsx

```diff
@@ -1,43 +1,34 @@
-import React, { PureComponent } from 'react'
+import React, { memo } from 'react'
 import PropTypes from 'prop-types'
 import { ButtonLink } from 'cozy-ui/transpiled/react/Button'
 import { translate } from 'cozy-ui/transpiled/react/I18n'
-import { queryConnect, Q } from 'cozy-client'
+import { withClient } from 'cozy-client'
+import useAppLinkWithStoreFallback from '../../hooks/useAppLinkWithStoreFallback'
 
-export class DriveLink extends PureComponent {
-  render() {
-    const { folderId, driveQuery, t } = this.props
-    let hasDrive = false
-    if (driveQuery.fetchStatus === 'loaded') {
-      if (driveQuery.data.length > 0) {
-        hasDrive = true
-      }
-    }
-    const href = hasDrive
-      ? `${driveQuery.data[0].links.related}#/files/${folderId}`
-      : undefined
+const DriveLink = memo(({ folderId, client, t }) => {
+  const { fetchStatus, url, isInstalled } = useAppLinkWithStoreFallback(
+    'drive',
+    client,
+    `#/files/${folderId}`
+  )
+  const href = fetchStatus === 'loaded' && isInstalled ? url : undefined
 
-    return (
-      <ButtonLink
-        href={href}
-        target="_parent"
-        disabled={!hasDrive}
-        subtle
-        icon="openwith"
-        label={t('account.success.driveLinkText')}
-      />
-    )
-  }
-}
+  return (
+    <ButtonLink
+      href={href}
+      target="_parent"
```

Comment:
BanksLinks doesn't have this attribute. Is it wanted?

Reply:
Same, let me check but my guess is it's not intentional.